Services/data/devicemodels/engine-01.json

```diff
@@ -53,10 +53,6 @@
             "Type": "javascript",
             "Path": "FirmwareUpdate-method.js"
         },
-        "Restart": {
```

Comment:
We are just hiding the functionality right now but not removing the actual file, right?

Reply:
I removed the method file too, as part of my previous change #201. The method (before my change) was only a TODO anyway.